workers.tf

```diff
@@ -143,18 +77,14 @@ resource "aws_autoscaling_group" "workers" {
       ],
       [
         for tag_key, tag_value in var.tags :
-        {
-          "key"                 = tag_key,
-          "value"               = tag_value,
-          "propagate_at_launch" = "true"
-        }
-        if tag_key != "Name" && !contains([for tag in lookup(var.worker_groups[count.index], "tags", local.workers_group_defaults["tags"]) : tag["key"]], tag_key)
+        map(
```

Comment:
Do we need something similar to ffdc7a79a575d0c1866bc97d3d7c804990cd66a9 here?

Reply:
:heavy_check_mark: